app/change_set_persisters/change_set_persister/create_tombstone.rb

```diff
@@ -8,7 +8,8 @@ def initialize(change_set_persister: nil, change_set:, post_save_resource: nil)
     end
 
     def run
-      return unless resource.is_a?(FileSet) && resource.try(:original_file) && parent
+      return unless resource.is_a?(FileSet) || resource.decorate.respond_to?(:file_sets)
```

Comment:
So this would not create a tombstone for any resource whose children are other resources? Like a multi-volume work. is that correct?

Reply:
I think we can change this to check if it should be preserved. Something like `DynamicChangeSet.new(resource).preserve?`